### PR TITLE
add pr template

### PR DESCRIPTION
**Pull request by burmudar**
Add a PR Template to add a test plan

### CODE REVIEW

**Review by doragrgic — APPROVED** [review]

File: .github/workflows/pr-auditor.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+on:
+  pull_request_target:
+    types: [ closed, edited, opened, synchronize, ready_for_review ]
+
+jobs:
+  check-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with: { repository: 'sourcegraph/sourcegraph' }
```
Comment (burmudar) [inline .github/workflows/pr-auditor.yml:12]: ah need to fix this

File: .github/workflows/pr-auditor.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  check-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with: { repository: 'sourcegraph/cody' }
+      - uses: actions/setup-go@v2
+        with: { go-version: '1.19' }
+
+      - run: ./dev/pr-auditor/check-pr.sh
```
Comment (burmudar) [inline .github/workflows/pr-auditor.yml:16]: woops 🔥

**Review by jhchabran — COMMENTED** [review]
Happy to approve, as it's useful to merge this asap, but I'm curious, why did we duplicate PR-Auditor's code? I understand that we may not want to clone sg/sg on every build, in which case, why don't we put it in a separate repo?

**Comment by burmudar** [comment]
> Happy to approve, as it's useful to merge this asap, but I'm curious, why did we duplicate PR-Auditor's code? I understand that we may not want to clone sg/sg on every build, in which case, why don't we put it in a separate repo? 

@jhchabran yup that was the initial _reason_ why it got duplicated and also why I was hesitant to merge since this felt like the wrong approach. I'll move it to a different repo today!

File: .prettierignore [hunk 3]
```diff
@@ -1,2 +1,4 @@
+.github/PULL_REQUEST_TEMPLATE.md
+ci/
```
Comment (burmudar) [inline .prettierignore:2]:
```suggestion
```

**Review by jhchabran — APPROVED** [review]

**Pull request merged**